app-misc/crunch/crunch-3.6.ebuild

```diff
@@ -13,14 +11,6 @@ LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~mips ~x86"
 
-src_prepare() {
-	epatch_user
-}
-
-src_compile() {
-	emake CC="$(tc-getCC)"
```

Comment:
sure?

Reply:
Nope, I misunderstood why it was needed. Thanks for catching :) got it explained on IRC. 
Will revert.